docs/troubleshooting.md

```diff
@@ -4,15 +4,32 @@ Troubleshooting
 Deleting a VFS for Git repo
 ---------------------------
 
-Since a VFS for Git clone has a running `GVFS.Mount` process to track the
-Git index and watch updates from the ProjFS filesystem driver, you must
-first run `gvfs unmount` before deleting your repository. This will also
-remove the repository from the auto-mount feature of `GVFS.Service`.
-
-If you have deleted the enlistment or its `.gvfs` folder, then you will
-likely see alerts saying "Failed to auto-mount at path `X`". To remove
-this enlistment from the auto-mount feature, remove the appropriate line
-from the `C:\ProgramData\GVFS\GVFS.Service\repo-registry` file.
+1. Un-mount the repo.
+
+    Since a VFS for Git clone has a running `GVFS.Mount` process to track the
+    Git index and watch updates from the ProjFS filesystem driver, you must
+    first run `gvfs unmount` before deleting your repository. This will also
+    remove the repository from the auto-mount feature of `GVFS.Service`.
+
+    Make sure the current working directory of your shell is not in the VFS for Git repo and that no other processes are using files in it. For example:
+
+    ```
+    C:\Users\you\big_repo\src\> cd ..\..
+    C:\Users\you\> gvfs unmount big_repo
+    ```
+
+    If you have deleted the enlistment or its `.gvfs` folder, then you will
+    likely see alerts saying "Failed to auto-mount at path `X`". To manually remove
+    this repo from the auto-mount feature, remove the appropriate line
+    from the `C:\ProgramData\GVFS\GVFS.Service\repo-registry` file.

```

Comment:
If you manually unmount the repo, then the repo should be removed from the repo-registry. Could this warning be placed outside of the bulleted list, to add that help only for users who tried deleting without following these instructions?

Reply:
Ya that makes sense, I thought this was sort of the existing "warning" that you shouldn't just delete it. We can make the an explicit warning at the top. If someone did try to delete it prematurely are the resolution steps to 
1. remove the auto-mount entry
2. finish actually deleting?

Reply:
Yes, that should work. It is possible that `GVFS.Mount.exe` is still running on their enlistment, preventing the `src` directory from being deleted. They need to go to Task Manager and find the `GVFS.Mount.exe` process with that enlistment in the "Command Line" column:

![image](https://user-images.githubusercontent.com/570044/89896047-c4bf0e00-dbaa-11ea-9567-d9d5bb3669ea.png)

Reply:
Aha, I'll include this. Thanks!